### PR TITLE
feat: Use graphql to query unranked list-items

### PR DESCRIPTION
**Pull request by payers1**
I'm pretty sure `ItemAddr.vue`, `ItemRanking.vue` and `List.vue` can all be deleted since they're not being used.

We definitely don't _need_ to use graphql

To test: https://pantry-list-frontend-tnctpaiekz.now.sh/#/

### CODE REVIEW

File: src/store/item-ranking.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ const mutations = {
   setRankedItems (state, items) {
     state.rankedItems = items
   },
-  setUnrankedItems (state, items) {
-    state.unrankedItems = items
+  setUnrankedItems(state, items) {
+    state.unrankedItems = items.data.items
```
Comment (tstodter) [inline src/store/item-ranking.js]: screen shot 2018-03-29 at 3 26 12 pm

Comment (tstodter) [reply src/store/item-ranking.js]: A crappy fix would be `state.unrankedItems = items.data ? items.data.items : items`

Comment (tstodter) [reply src/store/item-ranking.js]: This is when dragging something from unranked to ranked.

**Review by tstodter — APPROVED** [review]

**Review by ryanSN — APPROVED** [review]
LGTM

**Pull request merged**